tests/test_driver_info.py

```diff
@@ -0,0 +1,35 @@
+"""Test module for testing methods DriverInfo class from driver_info folder """
+import json
+from f1pystats.driver_info import DriverInfo
+
+
+class TestDriverInfo:
+    """Class containaing the functions for testing the DriverInfo class methods"""
+    
+    data = ""
+    with open("tests/test_data/first_3_drivers_info_2000.json", encoding='utf-8') as f:
+        data = json.load(f)
+        f.close()
+        
+    driver = DriverInfo(data)
+    
+    def test_driver_names(self):
+        """Test for the get_driver_names methods of DriverInfo"""
+        assert self.driver.get_driver_names() == ['Jean Alesi', 'Rubens Barrichello', 'Luciano Burti']
+    
+    def test_driver_dob(self):
+        """Test for the get_driver_dob method of DriverInfo"""
+        assert self.driver.get_drivers_dob() == ['1964-06-11', '1972-05-23', '1975-03-05']
+    
+    def test_driver_nationality(self):
+        """Test for the get_driver_nationality method of DriverInfo"""
+        assert self.driver.get_driver_nationality() == ['French', 'Brazilian', 'Brazilian']
```

Comment:
You are calling ```get_driver_nationality()```, but there is no such function in ```DriverInfo```.

```DriverInfo``` contains ```get_drivers_nationality()```. (NOTE the difference with ```drivers``` and ```driver```)

Reply:
You should determine if you will use ```get_driver...``` or ```get_drivers...``` prefix for both test functions.